media-libs/jbigkit/jbigkit-2.1.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="A lossless image compression library"
+DESCRIPTION="JBIG-KIT is a software implementation of the JBIG1 data \
+compression standard (ITU-T T.82), which was designed for bi-level image \
+data, such as scanned documents. This library is available in portable C \
+code. It is widely used in fax products, printer firmware, printer drivers, \
+document management systems and imaging software."
+HOMEPAGE="https://www.cl.cam.ac.uk/~mgk25/jbigkit/"
+COPYRIGHT="1995-2014 Markus Kuhn
+	1989, 1991 Jef Poskanzer"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://www.cl.cam.ac.uk/~mgk25/download/jbigkit-$portVersion.tar.gz"
+CHECKSUM_SHA256="de7106b6bfaf495d6865c7dd7ac6ca1381bd12e0d81405ea81e7f2167263d932"
+PATCHES="jbigkit-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 ?x86_64"
```

Comment:
@Begasus, if you add support for secondary architectures to this recipe, could you please include `?x86` as primary arch and `?x86_gcc2` as secondary arch?

In other words, instead of making this change:
```diff
 ARCHITECTURES="x86_gcc2 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
```
could you please make this one:
```diff
-ARCHITECTURES="x86_gcc2 ?x86_64"
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 ?x86"
```
or, if you notice that jbigkit builds fine with gcc5:
```diff
-ARCHITECTURES="x86_gcc2 ?x86_64"
+ARCHITECTURES="x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
```
This does not cost much, and it is better, imo, than simply dropping x86 as primary arch and/or x86_gcc2 as secondary arch.

* Having `?x86` as primary arch means: _"We don't want to upset users of 32 bit x86 processors"_.
* Not having it means _"We don't care if you are not running x86_64."_
* Having `?x86_gcc2` as secondary arch means: _"If we create an hybrid `x86_64`+`x86_gcc2` then running BeOS apps using x86_gcc2 secondary arch on x86_64 will become a possibility, with some hacks."_
* Not having it means: _"Once we drop x86_gcc2 you will have to keep an old x86_gcc2 system if you wish to run any BeOS app on Haiku."_

I guess that adding x86_gcc2 secondary arch to x86_64 would (probably) not be an easy task, and I also know that this might even never happen. But since having `?x86` in `ARCHITECTURES` and `?x86_gcc2` in `SECONDARY_ARCHITECTURES` costs nothing, I think we should have them.

Thanks (and sorry for discussing this in your PR :)

Reply:
No problem for the discussion ;) it helps to clear out things, ps recipe isn't tested yet for x86 and x86_64 (but that will be in a bit) :)